engine/debug/debug_party.asm

```diff
@@ -20,7 +12,9 @@ SetIshiharaTeam:
 	call AddPartyMon
 	jr .loop
 
-IshiharaTeam:
+DebugNewGameParty: ; unreferenced (non-debug)
+	; Then producer Tsunekazu Ishihara used an Exeggutor for play testing.
```

Comment:
I really think a claim like this should include the direct quote and URL citation that were originally on top of this file.

Reply:
How about this?
```suggestion
	; Exeggutor is the only debug party member shared with Red, Green, and Japanese Blue.
	; "Tsunekazu Ishihara: Exeggutor is my favorite. That's because I was always using
	; this character while I was debugging the program."
	; Source: https://web.archive.org/web/20000607152840/http://pocket.ign.com/news/14973.html
```